shopify/packages/shopify/shopify_orders.js

```diff
@@ -0,0 +1,83 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+const axios = require('axios');
+
+async function getRequest(url) {
+
+	return await axios.get(url)
+		.then((response) => { return response.data; })
+		.catch((error) => { return error.response.data; });
+}
+
+function isSecretMissing(secrets) {
+
+	var ret = '';
+	if (!secrets.shopifyKey) {
+		ret += 'Shopify API key not found!\n';
+	} if (!secrets.shopifyPassword) {
+		ret += 'Shopify password not found!\n';
+	} if (!secrets.shopifyHostname) {
+		ret += 'Shopify hostname not found!\n';
+	}
+	return ret;
+}
+
+async function _command(params, commandText, secrets = {}) {
+
+	const error = isSecretMissing(secrets);
+	if (error) {
+		return {
+			response_type: 'in_channel', // or `ephemeral` for private response
+			text: error
+		};
+	}
+
+	const url = `https://${secrets.shopifyKey}:${secrets.shopifyPassword}@${secrets.shopifyHostname}/admin/api/2020-01/orders.json`;
+	var data = await getRequest(url);
+
+	if (data.errors) {
+		return {
+			response_type: 'in_channel', // or `ephemeral` for private response
+			text: data.errors
+		};
+	}
+
+	var text = '';
+	data.orders.forEach(order => {

```

Comment:
How many orders can a single request contain? I think we probably need some kind of pagination since the orders might exceed the slack message size limit.

Reply:
Agree. May be we can show the latest 10 orders?

Reply:
Pick a default. Add pagination in a second PR. We will get user feedback on how to fine tune this.

Reply:
Okay, I will add it in and start at 10 for now